lib/ui/notifications.dart

```diff
@@ -87,39 +88,88 @@ class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
-                  Divider(
-                    height: 20,
-                    thickness: 2,
-                    color: Colors.black,
+                  Padding(
+                    padding: EdgeInsets.all(16.0),
+                    child: Divider(
+                      height: 1,
+                      indent: 10,
+                      endIndent: 10,
+                      thickness: 1,
+                      color: Color.fromRGBO(51, 51, 102, 1),
+                    ),
+                  ),
+                  Container(  //the following code is used for notification subscriptions on the manage tab, but placed here for testing purposes and layout
+                    height: 400,
+                    child: SingleChildScrollView(
+                      child: Column(
+                        children: [
+                          for (var item in _subscriptions)
+                            Row(
+                              mainAxisAlignment: MainAxisAlignment.center, //displaying notifications subscriptions for testing purposes only
+                              children: [
+                                SizedBox(
+                                  child: Text(item.keyword, style:TextStyle(color:Color.fromRGBO(51, 51, 102, 1),
+                                      fontSize: 18),) ,
+                                  width: 270,
+                                ),
+                                SizedBox(
+                                  child: OutlinedButton(
+                                    child: Text(
+                                      'Delete',
+                                      style: TextStyle(color: Colors.white,fontSize: 18),
+                                    ),
+                                    style: ButtonStyle(
+                                        backgroundColor:
+                                        MaterialStateColor.resolveWith(
+                                                (states) => Colors.red),
+                                        shape: MaterialStateProperty.all(
+                                            RoundedRectangleBorder(
+                                                borderRadius:
+                                                BorderRadius.circular(30)))),
+                                    onPressed: () {
+                                      removeSubscription(item.keyword);
+                                    },
+                                  ),
+                                )
+                              ],
+                            ),
+                        ],
+                      ),
+                    ),
                   ),
                 ],
               ),
             ),
             Container(
               child: Column(
                 children: [
-                  Row(children: [
+                  SizedBox(height: 10,),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
I used
mainAxisAlignment: MainAxisAlignment.spaceEvenly,
to look the same as the list of subscriptions or the alignment of the list should be changed to the center.